### PR TITLE
[Docs] Spend cap guide

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

New guide for spend cap

### CODE REVIEW

File: apps/docs/pages/guides/platform/spend-cap.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  title: 'Spend cap',
+  description: 'Learn about the cost control of Supabase.',
+}
+
+For managing cost control for each project, there is a Spend Cap to manage your usage and costs.
```
Comment (kiwicopple) [inline apps/docs/pages/guides/platform/spend-cap.mdx:8]:
```suggestion
Supabase offers a "Spend Cap" on each project to manage your usage and costs.
```

File: apps/docs/pages/guides/platform/spend-cap.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Each project's Spend Cap determines wether your project can exceed the free quota allowance of any paid [line item](#what-can-i-be-charged-for) in a given billing cycle.
```
Comment (kiwicopple) [inline apps/docs/pages/guides/platform/spend-cap.mdx:10]:
```suggestion
This Spend Cap determines whether your project can exceed the free quota allowance of any [line item](#what-can-i-be-charged-for) in a given billing cycle.
```

File: apps/docs/pages/guides/platform/spend-cap.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+You can monitor your usage and costs using the [Usage section of your project](https://app.supabase.com/project/_/settings/billing/usage) in the dashboard.
+This will show you how much you have spent so far in the current billing cycle, as well as an estimate of your expected total cost for the cycle based on your current usage.
+
+## How can I turn the Spend Cap on or off?
+
+To set up a Spend Cap in your project, navigate to the [Subscription section](https://app.supabase.com/project/_/settings/billing/subscription) of your project.
+From there, you can click on "Change subscription" to select the plan you want to use and, at the same time, turn on or off the Spend Cap.
+
+## What happens when the Spend Cap is on
```
Comment (kiwicopple) [inline apps/docs/pages/guides/platform/spend-cap.mdx:37]:
```suggestion
## What happens when the Spend Cap is on?
```

File: apps/docs/pages/guides/platform/spend-cap.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Once you use all your free quota allowance for a [line item](#what-can-i-be-charged-for), your project will not allow any further usage on that line item until the next billing cycle begins.
+
+This can help you to avoid unexpected costs and manage your expenses more effectively.
+
+## What happens when Spend Cap is off?
+
+Once you use all the free quota allowance for a [line item](#what-can-i-be-charged-for), your project will continually be able to make further usage.
```
Comment (kiwicopple) [inline apps/docs/pages/guides/platform/spend-cap.mdx:45]:
```suggestion
Your project will continue to operate after using all the free quota allowance for each [line item](#what-can-i-be-charged-for).
```

File: apps/docs/pages/guides/platform/spend-cap.mdx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+This additional usage will be charged based on the line item's cost per unit outlined on the [pricing page](/pricing).
+
+<Admonition type="note">
+  It is advised, when your Spend Cap is **off**, to monitor your usage and costs using the [Usage
```
Comment (kiwicopple) [inline apps/docs/pages/guides/platform/spend-cap.mdx:49]:
```suggestion
  When your Spend Cap is **off**, we recommend monitoring your consumption and costs using the [Usage
```

File: apps/docs/pages/guides/platform/spend-cap.mdx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+## What additional usage can I be charged for?
+
+Supabase charges for certain usage line items in your project. Those line items include:
+
+- Database Size
+- Storage Size
+- Egress
+- Asset transformations
+- Realtime connections
```
Comment (kevcodez) [inline apps/docs/pages/guides/platform/spend-cap.mdx:21]:
```suggestion
```

We only have the peak connections as a line item, not the connections itself

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 7]
```diff
@@ -674,6 +674,11 @@ export const platform = {
         { name: 'SSL Enforcement', url: '/guides/platform/ssl-enforcement', items: [] },
       ],
     },
+    {
+      name: 'Billing Management',
```
Comment (kevcodez) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:678]:
```suggestion
      name: 'Billing',
```

Up to you though, I'd prefer the short version

**Review by kevcodez — CHANGES_REQUESTED** [review]
Just one small change and a nit - great to get some docs for billing 👏

**Review by kevcodez — APPROVED** [review]

**Pull request merged**